epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -154,6 +164,24 @@ public synchronized void init(ProcessingEnvironment processingEnv) {
     return types;
   }
 
+  @Override
+  public Set<String> getSupportedOptions() {
+    Set<String> options = new LinkedHashSet<>();
+    options.add(PROCESSOR_OPTION_IMPLICITLY_ADD_AUTO_MODELS);
+    options.add(PROCESSOR_OPTION_VALIDATE_MODEL_USAGE);
+    options.add(PROCESSOR_OPTION_REQUIRE_ABSTRACT_MODELS);
+    options.add(PROCESSOR_OPTION_REQUIRE_HASHCODE);
+    options.add(PROCESSOR_OPTION_DISABLE_KOTLIN_EXTENSION_GENERATION);
+    if (trees != null) {
+      IncrementalAnnotationProcessorType incapType = IncrementalAnnotationProcessorType.ISOLATING;
+      if (!configManager.disableKotlinExtensionGeneration()) {
+        incapType = IncrementalAnnotationProcessorType.AGGREGATING;
```

Comment:
Why do Kotlin extensions make the processor aggregating? There's one extension file per model. Seems like an isolating use case to me.

Reply:
This is because it's only generating one source file for all the aggregated extensions and originating elements sent to the filer are reported on a per-file basis. I agree isolating would be better though, but think it would be better to change that in a separate PR to limit scope. It should be possible to do that in a non-source-breaking way via just generating them into multiple files.